### PR TITLE
Additional PoS Polish

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

PoS

Move FC's office to give him more space and be next to CO.

Replace old FC's office into chapel.

Fix space tiles in req. Yeah, there were space tiles despite sprites saying otherwise. It's a weird bug.

Expand briefing and make TGMC letters go underneath chairs.

Add another grinder for chem users to use.

## Why It's Good For The Game

Space tiles really bad. Damn, map conflicts.

Since briefing is added, I removed the office briefing space that was next to CO, replaced that with FC's office, and make chapel.

RipGrayson gave me code to make TGMC letters in briefing go underneath chairs.

## Changelog
:cl:
add: Expand PoS' briefing and make TGMC letters go underneath chair
add: another grinder for chem users to use in PoS
add: Move FC's office to give him more space and be next to CO.
add: Replace old FC's office into chapel.
fix: space tiles in PoS' req attempt 2
fix: rogue piping
/:cl:

### CODE REVIEW

**Review by BraveMole — APPROVED** [review]

**Comment by RipGrayson** [comment]
Maintainers please don't merge this PR until either or Elsa can confirm that the space tiles are truly gone. There's an issue where the space tiles are invisible to map editing tools but still exist in game (which is true of the latest commit).

**Review by RipGrayson — APPROVED** [review]
As far as I can tell the space tiles have finally been annihilated, this PR is fine
EDIT: Hold up there's still invisible space tiles in req
EDIT EDIT: Apparently the space tiles in req do not actually exist as confirmed by manually inspecting the map file, this PR is fine again.

**Pull request merged**
